app/views/devise/registrations/new.html.erb

```diff
@@ -1,29 +1,34 @@
-<h2>Sign up</h2>
-
+<div class ="view-container">
 <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
   <%= render "devise/shared/error_messages", resource: resource %>
+  <div class="mb-3">
+     <div class="form-contro"><%= f.text_field :name, autofocus: true, autocomplete: "name", placeholder: "Name"%></div>
+  </div>
 
-  <div class="field">
-    <%= f.label :email %><br />
-    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+   <div class="mb-3">
+   <div class="form-contro"> <%= f.text_field :bio, autofocus: true, autocomplete: "bio", placeholder: "Bio"%></div>
   </div>
 
-  <div class="field">
-    <%= f.label :password %>
+  <div class="mb-3">
+     <div class="form-contro"><%= f.email_field :email, autofocus: true, autocomplete: "email", placeholder: "Email"%></div>
+  </div>
+
+  <div class="mb-3">
+    <div class="form-contro"> <%= f.label :password %><div>
     <% if @minimum_password_length %>
     <em>(<%= @minimum_password_length %> characters minimum)</em>
     <% end %><br />
```

Comment:
- [ ] Please make sure to structure the design to look like the given wireframe for the login and sign-up pages.

![image](https://github.com/H-Ramalan/blog/assets/62290545/ac24ab13-9758-4c71-b1e9-4a643ace8937)